src/_macosx.m

```diff
@@ -392,6 +385,9 @@ static CGFloat _get_device_scale(CGContextRef cr)
 static PyObject*
 FigureCanvas_flush_events(FigureCanvas* self)
 {
+    // We need to interrupt the runloop very briefly to allow the view to be
```

Comment:
Is this interrupting the runloop (which is what I assume that OSX calls the event loop?) or letting it actually run (and interrupting the Python interpreter)?

Reply:
I'm not certain... Is there a way to debug that and see what is going on? I think you are right though, the text I had was misleading about "interrupting". I'm pretty sure we are doing the latter, so I updated the comment.